### ISSUE TITLE
Mount Issue

### ISSUE THREAD

**Issue by katech725** [ynazar1/Arh#10]
The mount up (Mount Green, Yellow, Red) feature is not working when adding different color cones to the HUD. No errors or lua messages - simply nothing happens. Disabled all other addons.

PS- thank you so much for keeping this amazing addon alive - it turns archaeology from a C- activity to an A+. You rock. :)

**Comment by ynazar1** [comment]
Hi. Thank you for your appreciation. Do you have any viable mounts set as your favorites? IIRC it chooses from the favorite mounts only. 

Edit: I was right:

`        (C_MountJournal.Summon or C_MountJournal.SummonByID)(0) -- random favorite mount`

**Comment by katech725** [comment]
Thanks for checking on this! I did not have any favorites set. I went and added 3 favorite mounts to the journal, disabled all other addons, renamed WTF and Cache and completely restarted game. Same results as before, no mount up when adding the cones.
As an add, I main druid and so the addon would typically just trigger shapeshift to flight form vs actually mounting. Out of curiosity, I tried this on a non-Druid character and it works! A random favorite mount is summoned when adding the cones. Only the druid character is experiencing the issue.
